components/home/partners/Partners.tsx

```diff
@@ -0,0 +1,20 @@
+import { Bloq } from "./logos/Bloq";
+import { Chipax } from "./logos/Chipax";
+import { MindfulChef } from "./logos/MindfulChef";
+import { OneMedical } from "./logos/OneMedical";
+import { Raisedev } from "./logos/Raisedev";
+import styles from "./Partners.module.scss";
+
+export const Partners = () => (
+  <div
+    className={styles.partners + " d-flex flex-column justify-content-center"}
+  >
+    <div className="d-flex justify-content-around">
```

Comment:
A few of these look stretched (Bloq). Maybe something about the flex?
![2021-12-14_12-55-06](https://user-images.githubusercontent.com/633226/146078278-3c159d24-4ad6-43e6-8a06-c362dff2add9.png)

Reply:
These logos need a second pass, currently the bad ones are png's embedded into svg's and that's bad all around.
I'm planning to wait until we know which we want here to do another pass.

Reply:
Gotcha. I removed one and shuffled the order. Logo approvals can take time, so I say let's move forward with these.

Reply:
Adding/iterating on these in the future shouldn't be a big deal, I hope.

Reply:
It won't be!